tests/test_dynamodb.py

```diff
@@ -0,0 +1,125 @@
+from pytest_dbfixtures import factories
+
+dynamodb_proc = factories.dynamodb_proc(
+    jar_path='/tmp/dynamodb/DynamoDBLocal.jar'
```

Comment:
Yeah, you have to provide a path to the jar file. I don't think it is a good idea to try finding this file